frontend/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.tsx

```diff
@@ -446,7 +451,7 @@ function makeMarkdownHeading(tag: string, markdown: string): string {
 }
 
 export function Heading(props: HeadingProtoProps): ReactElement {
-  const { width } = props
+  const { width, element } = props
   const { tag, anchor, body } = props.element
```

Comment:
Nit: could take advantage of the prop.element deconstruction to access help directly 
```
const { width } = props
const { tag, anchor, body, help } = props.element
```

Reply:
Yup, this will improve this, thank you it's updated 👍